src/test/java/htmlFindElements.java

```diff
@@ -0,0 +1,76 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class htmlFindElements {
+    WebDriver driver;
+    private static final String Mysite = "C:\\Users\\Бубята\\IdeaProjects\\HerokuApp\\src\\test\\resources\\Test.html";
+
+    @BeforeMethod
+    public void initMethod() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+    }
+    @Test
+    public void htmlFindElementTable() {
+        driver.get(Mysite);
+        String actualResult = driver.findElement(By.id("table01")).getAttribute("id");
+        Assert.assertEquals(actualResult, "table01");
+        driver.quit();
+    }
+
+    @Test
+    public void htmlFindElementInput() {
+        driver.get(Mysite);
+        String actualResult = driver.findElement(By.name("email")).getAttribute("name");
+        Assert.assertEquals(actualResult, "email");
```

Comment:
here you're also checking html attribute, please fix it

Reply:
done